besser/bot/platforms/payload.py

```diff
@@ -20,6 +23,9 @@ class PayloadAction(Enum):
     BOT_REPLY_STR = 'bot_reply_str'
     """PayloadAction: Indicates that the payload's purpose is to send a bot reply containing a :class:`str` object."""
 
+    BOT_REPLY_FILE = 'bot_reply_file'
+    """PayloadAction: Indicates that the payload's purpose is to send a bot file containing a :class:`dict` object."""
```

Comment:
I would replace by: PayloadAction: Indicates that the payload's purpose is to send a bot reply containing a file.

Is the file a dict? then, is it specified anywhere the structure of the dict? (name, type, content...)